src/shims/unix/linux/fd/event.rs

```diff
@@ -13,7 +14,9 @@ use std::io;
 /// <https://man.netbsd.org/eventfd.2>
 #[derive(Debug)]
 pub struct Event {
-    pub val: u32,
+    /// The object contains an unsigned 64-bit integer (uint64_t) counter that is maintained by the
+    /// kernel. This counter is initialized with the value specified in the argument initval.
+    pub val: Cell<u64>,
```

Comment:
But what does the value *mean*? This documentation is rather mysterious.

Reply:
The original POSIX documentation is not very clear either ^^ the value can mean all kinds of things depending on the usage pattern. All that the storage should know is that it's a 64 bit int

Reply:
Might be good to have an example then. POSIX docs being bad is no excuse for our own docs being bad. ;)